tree/20_cfe_basics/dry_run.cf

```diff
@@ -283,40 +283,6 @@ body action databases_action
     action_policy => "fix";
 }
 
-# in bundle edit_line
```

Comment:
could you add a comment explaining why these aren't there? Could be in the comment explaining why other types are already skipped (line 191)